USER_GUIDE.md

```diff
@@ -439,6 +440,51 @@ print('\nSearch results:')
 print(response)
 ```
 
+## Using IAM authentication with Async opensearch-py
+
+Make sure to use the new `RequestsAsyncHttpConnection` connection class with the new async `AWSV4SignerAuthAsync` signer.
```

Comment:
Nit: Since it's a user guide I would remove "new". So just "Use `Request...` with the async `AWSV...` signer."

Reply:
Makes sense. Fixed.